src/server/utilities/mvtHeader/getMvtVaryHeaders/index.js

```diff
@@ -0,0 +1,21 @@
+import enabledExperimentList from '../enabledExperimentsList';
+
+const getMvtVaryHeaders = (mvtExperiments, service, derivedPageType) => {
+  return enabledExperimentList
+    .reduce((headerNames, experiment) => {
+      const { name, services, pageTypes } = experiment;
+
+      const isMvtExperimentEnabled =
+        mvtExperiments.some(({ experimentName }) => experimentName === name) &&
+        services.includes(service) &&
+        pageTypes.includes(derivedPageType);
```

Comment:
Thinking of moving this logic to the `getMvtExperiments` logic, as at the moment the components have to state what service the experiment is running, whereas if we move this logic and add an enabled key to the experiment object with a boolean value, then we can just use that within here to work out which headers to vary on, and the components can also use that boolean to know if they should show the experiment or not.